[BUG] public main compile error

Javac complains about not finding the symbol string

> d:\Github\LS4P\server\out\compile\ProcessingDefault.java:46: error: cannot find symbol
public static void main(string[] args) {
                        ^
  symbol:   class string
  location: class ProcessingDefault
1 error

Because lowercase string isn't a java type, was introduced during PR #24 but wasn't noticed sooner because it only started complaining after removing a lot of test code

**Solution:**

--- a/server/src/grammer/terms/preprocessingsnippets.ts
+++ b/server/src/grammer/terms/preprocessingsnippets.ts
@@ -160,7 +160,7 @@ ${settingsContext}
 
 function preprocessingFooter(): string{
 	let generatedFooter: string = `
-public static void main(string[] args) {
+public static void main(String[] args) {
 PApplet.main("${defaultClassName}");
 }`
 	return generatedFooter

